hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -3,18 +3,1146 @@
+    }
+
+    // KEYMGR (pre-verified IP) integration tests:
+
+    // OTBN (pre-verified IP) integration tests:
+    {
+      name: chip_otbn_op
+      desc: '''Verify an OTBN operation.
+
+            SW test directs the BIGNUM engine to perform an operation. The BIGNUM SW image is
+            backdoor loaded into OTBN IMEM. SW validates the reception of the otbn done interrupt.
+            SW also verifies the correctness of the OTBN operation using a reference model. Details
+            TBD.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_otbn_imem_uncorrectable_alert
+      desc: '''Verify the imem uncorrectable alert from OTBN.
+
+            Inject 2 bit errors within the IMEM SRAM inside OTBN via backdoor and ensure that this
+            alert propagates to an NMI.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_otbn_dmem_uncorrectable_alert
+      desc: '''Verify the dmem uncorrectable alert from OTBN.
+
+            Inject 2 bit errors within the DMEM SRAM inside OTBN via backdoor and ensure that this
+            alert propagates to an NMI.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_otbn_reg_uncorrectable_alert
+      desc: '''Verify the reg uncorrectable alert from OTBN.
+
+            Details TBD. Ensure that this alert propagates to an NMI.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_otbn_mem_encr
+      desc: '''Verify the encryption of the mem within OTBN using the key provided by the OTP.
+
+            Details TBD.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_otbn_idle
+      desc: '''Verify OTBN idle signaling to clkmgr.
+
+            Details TBD.
+            '''
+      milestone: V2
+      tests: []
+    }
+
+    ////////////////////////////////////////////////
+    // Memory & Controllers                       //
+    // ROM, RAM, FLASH, FLASH_CTRL, OTP, OTP_CTRL //
+    ////////////////////////////////////////////////
+
+    // ROM (pre-verified IP) integration tests:
+    {
+      name: chip_rom_access
+      desc: '''Verify access to the rom.
+
+            Verify that the CPU can fetch instructions from the ROM. Nothing extra needs to be done
+            here - all SW tests do this anyway.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_rom_security_features
+      desc: '''Verify ROM security / ECC features if available.
+
+            TBD.
+            '''
+      milestone: V2
+      tests: []
+    }
+
+    // SRAM (pre-verified IP) integration tests:
+    {
+      name: chip_sram_access
+      desc: '''Verify access to the SRAM.
+
+            Verify that the CPU can fetch data from the SRAM. Nothing extra needs to be done
+            here - all SW tests do this anyway.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_sram_encr
```

Comment:
just a nit, we've been consistently avoiding the term "encrypted" and using "scrambled".
Since encrypted triggers a different standard.

Reply:
Ok, updated.